<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pievienot_ligums</class>
 <widget class="QDialog" name="pievienot_ligums">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>255</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pievienot līgumu</string>
  </property>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Līguma nummurs </string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pievienot_lig">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>pievienot</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="mosl_datums">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>121</width>
     <height>26</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2019</year>
     <month>5</month>
     <day>1</day>
    </datetime>
   </property>
   <property name="displayFormat">
    <string>dd.MM.yyyy</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Līgums</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Noslēgšanas datums</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lig_nr">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Slodze (h)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Datums līdz</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDateEdit" name="mosl_datums_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>121</width>
     <height>26</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2019</year>
     <month>5</month>
     <day>1</day>
    </datetime>
   </property>
   <property name="minimumTime">
    <time>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
    </time>
   </property>
   <property name="displayFormat">
    <string>dd.MM.yyyy</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>48</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
